content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -314,18 +315,17 @@ Server:  Message Received
 ```
 4.The server receives and simultaneously shows;
 
-```bash
+```
 Connected
 Client: Hey there 
 Client: am doing this for the first time
 Client: BYE
 Closing connection
 ```
  Important note
```

Comment:
Change:

>Important note
Port number may fail to work you can always change it and use a different one from those commercially owned.


To:

>>Important note: The port number you are using may cause an error if the port is already in use. To fix this, change the port number to a value that is not a commercially owned port.

Fixed grammar, formatting, and sentence flow.